searchlib/src/vespa/searchlib/features/utils.cpp

```diff
@@ -1,28 +1,43 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 #include "utils.hpp"
+#include <charconv>
 
 namespace search::features::util {
 
+template <typename T>
+T strToInt(vespalib::stringref str)
+{
+    T retval = 0;
+    if ((str.size() > 2) && (str[0] == '0') && ((str[0] | 0x20) == 'x')) {
+        std::from_chars(str.data(), str.data()+str.size(), retval, 16);
+    } else {
+        std::from_chars(str.data(), str.data()+str.size(), retval, 10);
```

Comment:
Unless we know that the input is well formed, consider checking the returned `std::from_chars_result` for errors.

Reply:
The usage this one is best effort using 0 as fallback in case of error.